src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfacePrompt.cs

```diff
@@ -322,24 +322,12 @@ public override
                 // mind is lost. This can lead to a malicious
                 // server prompting the user for a request
                 // which can appear to come from locally.
-                if (!PromptUsingConsole() && desc.ModifiedByRemotingProtocol)
```

Comment:
Deleting this code makes the comment above very confusing.
It also implies you are introducing a potential security risk.

Reply:
Will remove the comment since it only applied to the GUI prompt.